src/components/blocks.ts

```diff
@@ -143,6 +145,32 @@ export default class Blocks {
     this.blocks[first] = secondBlock;
   }
 
+  /**
+   * Move a block from one to another index
+   * @param {Number} toIndex - new index of the block
+   * @param {Number} fromIndex - block to move
+   */
+  public move(toIndex: number, fromIndex: number): void {
+    const block = this.blocks.splice(fromIndex, 1)[0];
```

Comment:
Is there any reason to use `splice` instead of `this.blocks[fromIndex]`?

Reply:
As discussed, I added some notes here https://github.com/codex-team/editor.js/pull/1007/commits/1551be3a2f46f97233004e1396376a2353d02ab2#diff-a1e3d44548903047b0a0d542527a389cR154-R158.